watcher_latency.go

```diff
@@ -56,10 +60,25 @@ func (lw *LatencyWatcher) refresh(o *Observer, infoKeys []string, rawMetrics map
 		}
 	}
 
+	// loop all the latency infokeys
+	for ik := range infoKeys {
```

Comment:
This can be simplified by directly accessing values in loop. 

for _, key := range infoKeys {
...
use the key directly instead of accessing via the index again.